### PR TITLE
Fixed issues mentioned by Abijeet in the review.

### PR DESCRIPTION
**Pull request by prasanna03**

### CODE REVIEW

File: Common/data/data.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "data": [
     {
-      "desc": "<ul><li>A responsive page that uses the bootstrap classes.</li><li>Add/update/delete tasks and view the status of the tasks in the charts shown in the page.</li><li>It uses CakePHP API and returns JSON which contains list of tasks.</li></ul>",
+      "desc": "<ul><li>A very basic task management system with a responsive design.</li><li>Initial data is fed through a CakePHP REST API.</li></ul>",
```
Comment (AbijeetP) [inline Common/data/data.json]: This is not a list.

File: Common/js/script.js [hunk 2]
```diff
@@ -21,14 +61,21 @@ $(document).ready(function () {
         "targets": 1
       }, {
         "render": function (data, type, row) {
```
Comment (AbijeetP) [inline Common/js/script.js]: please remove quotes for properties ... everywhere.

File: Common/data/data.json (hunk 1 above)
Comment (prasanna03) [reply Common/data/data.json]: Okay, I will do that.

File: Common/js/script.js (hunk 2 above)
Comment (prasanna03) [reply Common/js/script.js]: Okay, I will do that.

**Pull request merged**